eval/measure_judged.py

```diff
@@ -64,12 +64,20 @@ def main():
     parser.add_argument('--cutoffs', metavar='N', nargs='+', type=int, default=[10, 100, 1000],
                         help='Space-separate list of cutoffs, e.g., --cutoffs 10 100 1000.')
     parser.add_argument('--q', '-q', action='store_true', dest='print_topic', help='Print metrics per topic.')
+    parser.add_argument('--topics-in-qrels-only', dest='valid_topics', action='store_true', help='Ignore unlisted topicIds in qrels')
```

Comment:
to avoid confusion, better to use the same name? i.e, remove `dest='valid_topics'` and use args.topics_in_qrels_only in the python code?